README.md

```diff
@@ -15,7 +15,8 @@ If you have just started learning about self-sovereign identity, here are some r
 
 * This extended tutorial introduces Indy, explains how the whole ecosystem works, and how the
 functions in the SDK can be used to construct rich clients: [Indy-SDK Getting-Started Guide](docs/getting-started/indy-walkthrough.md)
-
+**Please take note** that this tutorial doesn't contain a part related to peer-to-peer interaction. 
```

Comment:
I find this comment extremely confusing. Alice and Faber and Thrift are all peers. How can we say that our tutorial doesn't cover peer-to-peer interactions, then? What do we mean? We need probably 2-3 sentences more of explanation that contrasts what the tutorial does cover with what it does not cover. Perhaps something like this: "This tutorial focuses on an interaction between an individual and several institutions. Institutions register DIDs on a ledger so they can issue credentials. We won't cover how Alice could connect to another private individual. That involves the use of peer DIDs [link to peer did spec] and is explored in greater detail in Aries."

Reply:
Thank you. I agree. it is really confusing. You didn't understand what I wanted to say and suggested a variant from different area (I think).
What I wanted to say: 
   The guide included a suggestion on how sides can exchange DIDs, encode and exchange messages. 
  We deleted this parts in favor of Aries project. 

I updated the phrase. Could you look at a new variant?

```
**Please take note** that we are not going to cover how sides set up connection and interact with each other as part of this tutorial.
We assume that already there is established some communication channel between each pair and this channel can be used for message exchange.
Instead of message sending we will just use construction like:
```
    faber['alice_reply'] = alice['reply']
    alice['faber_response'] = faber['response']
```

How this communication channel can be built you can find at [Aries](https://github.com/hyperledger/aries) project which describes it in great details.
```

Reply:
Yes, the new update makes a lot more sense. Thanks.